src/qml/MessageAlertBubble.qml

```diff
@@ -30,9 +30,9 @@ ListItemWithActions{
 
     readonly property var messageStatus: messageData.textMessageStatus
 
-    readonly property string permanentErrorText: i18n.tr("Oops, there has been an error with the MMS system and this message could not be retrieved. Please ensure Cellular Data is ON and MMS settings are correct, then ask the sender to try again.")
-    readonly property string temporaryErrorText: i18n.tr("Oops, there has been an error with the MMS system and this message could not be retrieved. Please ensure Cellular Data is ON and MMS settings are correct, then tap the redownload button to try to retrieve the message again.")
```

Comment:
"Oops, " is a spliced sentence.
Cellular data is not a proper noun, and it is not quoted.
"ON" is an error.